### ISSUE TITLE
Can we provide wrappers and examples for all the org and customer specific endpoints for enterprises?

### ISSUE THREAD

**Issue by dmiddlecamp** [spark/particle-api-js#23]
Received this as a feature request from a customer, I think right now this library is fairly user-centric, and doesn't provide features for Orgs.  Happy to answer any questions offline if I can, thank you!

**Comment by monkbroc** [comment]
I agree this is needed. As a side note, I plan on sprucing up the README to include a getting started section and links to examples in the docs.

**Comment by jme783** [comment]
This would be awesome! 👍
